app/views/handlers/submission.py

```diff
@@ -34,6 +34,7 @@ def submit_questionnaire(self):
         if not submitted:
             raise SubmissionFailedException()
 
+        cookie_session["display_address"] = metadata.get("display_address")
```

Comment:
The card specifies the address should be stored in the session rather than the cookie. Although not implemented yet, storing this in the cookie doesn't align with the decisions made in https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/doc/architecture/decisions/0010-cookies.md.